zetaclient/chains/ton/liteapi/client_live_test.go

```diff
@@ -0,0 +1,82 @@
+package liteapi
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tonkeeper/tongo/config"
+	"github.com/tonkeeper/tongo/liteapi"
+	"github.com/tonkeeper/tongo/ton"
+	"github.com/zeta-chain/node/zetaclient/common"
+)
+
+func TestClient(t *testing.T) {
+	if !common.LiveTestEnabled() {
+		t.Skip("Live tests are disabled")
+	}
+
+	var (
+		ctx    = context.Background()
+		client = &Client{Client: mustCreateClient(t)}
+	)
+
+	t.Run("GetFirstTransaction", func(t *testing.T) {
```

Comment:
It would be good to have a test for an account with no TXS.

Reply:
https://github.com/zeta-chain/node/pull/2896/commits/3f8f98958819c72911d6a78660c10490c14e72c6